docs/index.rst

```diff
@@ -0,0 +1,15 @@
+Welcome to LIVE DATA SERVER's documentation!
+====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   developer/index
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
```

Comment:
The module index link is not working, but we can also leave it as a reminder of something to do in the future

Reply:
I'll comment it out with a one-liner explanation